.NET/Microsoft.Recognizers.Text.DateTime/Extractors/BaseDateExtractor.cs

```diff
@@ -322,5 +329,30 @@ private List<Token> DurationWithBeforeAndAfter(string text, DateObject reference
             return ret;
         }
 
+        private void RemoveMoreThanOrLessThanPrefix(ExtractResult er)
+        {
+            var match = config.MoreThanRegex.Match(er.Text);
+            if (match.Success)
+            {
+                var originalLength = er.Text.Length;

```

Comment:
Nit: Simple refactor to reuse util code.

Reply:
Didn't find utility code doing similar thing.
Create a simple method to replace text using certain Regex and change corresponding files in ExtractResult, re-use this simple method to do the refactor.